Tribler/Test/Community/Multichain/StatisticsDispaly/test_statistics_database.py

```diff
@@ -0,0 +1,73 @@
+from Tribler.Test.test_as_server import BaseTestCase
+from Tribler.community.multichain.StatisticsDisplay.networkNode import NetworkNode
+from Tribler.community.multichain.StatisticsDisplay.dbDriver import DbDriver
+
+from twisted.internet.defer import inlineCallbacks
+from Tribler.dispersy.util import blocking_call_on_reactor_thread
```

Comment:
Please make sure to follow [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports) regulations as far as imports go.